dist/style.css

```diff
@@ -352,24 +336,18 @@ footer span {
     padding-top: 5rem;
   }
 
-  footer {
-    font-size: 0.4rem;
-    padding: 2.1rem;
-  }
   .footer {
-    padding: 2.1rem 10vw;
-    gap: 0.5rem;
+    font-size: 0.5rem;
   }
   footer img {
-    width: 19vw;
+    width: 18vw;
   }
   #email {
     justify-content: flex-start;
-    margin: 0;
   }
 }
 /* Mobile */
-@media only screen and (max-width: 480px) {
+@media only screen and (max-width: 580px) {
```

Comment:
than should use 580px for index page too
the behavior must be same at all pages

Reply:
well I played with the responsive yesterday. didn't like the way it works with 480px but wasn't sure it is ok to change it without your confirmation. I still do not like how the index page is working now. But let Lesha take a look at it first. I will fix for now to 580px

Reply:
kk